distribution/docs/src/main/resources/content/_appReferences/mg-spatial.adoc

```diff
@@ -9,29 +9,26 @@ The ${ddf-spatial} Application provides KML transformer and a KML network link e
+The `gazetteer` commands provide the ability to interact with the local GeoNames metacard collection in the core catalog. These GeoNames metacards are used by the `offline-gazetteer` feature, which is an optional feature available in this application and is explained above. Note that these commands are only available if the `offline-gazetteer` feature is installed.
 
-.GeoNames Command Descriptions
+.Gazetteer Command Descriptions
 [cols="2,8a,1m" options="header"]
 |===
 |Command
 |Description
 |Usage
 
 |`update`
-|Adds new entries to an existing local GeoNames index. Entries can be manually downloaded from http://download.geonames.org/export/dump, where the absolute path of the file would be passed as an argument to the command (ex. /Users/johndoe/Downloads/AU.zip). Currently .txt and .zip files are supported for manual entries. Entries can also be automatically downloaded from http://download.geonames.org/export/dump by passing the country code as an argument to the command (ex. AU) which will add the country to the local GeoNames index. The full list of country codes available can be found in http://download.geonames.org/export/dump/countryInfo.txt. Using the argument "all" will download all of the current country codes (this process may take some time). In addition to country codes, GeoNames also provides entries for cities based on their population sizes. The arguments "cities1000", "cities5000", and "cities15000" will add cities to the index that have at least 1000, 5000, or 15000 people respectively.
+|Adds new GeoNames metacards to the core catalog. Entries can be manually downloaded from http://download.geonames.org/export/dump, where the absolute path of the file would be passed as an argument to the command (ex. /Users/johndoe/Downloads/AU.zip). Currently .txt and .zip files are supported for manual entries. Entries can also be automatically downloaded from http://download.geonames.org/export/dump by passing the country code as an argument to the command (ex. AU) which will add the country as GeoNames metacards in the core catalog. The full list of country codes available can be found in http://download.geonames.org/export/dump/countryInfo.txt. Using the argument "all" will download all of the current country codes (this process may take some time). In addition to country codes, GeoNames also provides entries for cities based on their population sizes. The arguments "cities1000", "cities5000", and "cities15000" will add cities to the index that have at least 1000, 5000, or 15000 people respectively.
 
-The index location can be configured via the ${admin-console}. By default, the index location is `data/geonames-index`. If you specify a relative path, it is relative to the location of the unzipped
```

Comment:
❓ Should this be moved up under the `Offline Gazetteer Service` section rather than deleted altogether?

Reply:
I don't think this configuration option will be available after https://codice.atlassian.net/browse/DDF-3337 is merged since the GeoNames data will be stored in catalog core. @bdeining is my understanding correct?

Reply:
That's correct.  https://github.com/codice/ddf/pull/2420 renders the index location configuration obsolete since we are moving gazetteer support to Solr.